notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,757 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.      ax.plot(muTrue[0], muTrue[1], "x", linewidth=4, markerfacecolor="black", color="k",label='z')
</pre><p>What it is need of to add argument <code>linewidth=4? </code> Because now linewidth=4 will be applied in latexified and default (notebook) version.  if you observe latexified figure, the plot points look too much large than usual. Hence, we should not change default setting in latexified figute until it is required. </p><p> </p><p>You might think that, if you will not add <code>linewidth=4</code> then dots in default figure in notebook  looks very small, in that case you can use method <code>is_latexify_enabled()</code> from <code>probml_utils</code>.</p><p> </p><p>You can write like below:</p><pre class="ql-syntax" spellcheck="false">LINE_WIDTH = None if is_latexify_enabled() else 4

ax.plot(muTrue[0], muTrue[1], "x", linewidth=LINE_WIDTH, markerfacecolor="black", color="k",label='z')
</pre><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>